src/qml/controls/ToggleButton.qml

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2023 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+Button {
+    property color defaultColor: Theme.color.neutral2
+    property color hoverColor: Theme.color.neutral2
+    property color activeColor: Theme.color.neutral5
+    property color textColor: Theme.color.neutral7
+    property color textHoverColor: Theme.color.orangeLight1
+    property color textActiveColor: Theme.color.neutral9
+    id: root
+    checkable: true
+    hoverEnabled: true
```

Comment:
This is true by default for `Button`
```suggestion
```

Reply:
I noticed this on your commit.  It is not on by default in my Ubuntu or Fedora machines.